spec/models/user_spec.rb

```diff
@@ -6,12 +6,14 @@
   describe 'validations' do
     subject { user.valid? }
 
-    let(:user) { build(:user, email:, password:, name:, birth:, username:) }
+    let(:user) { build(:user, email:, password:, name:, birth:, username:, bio:, website:) }
     let(:name) { 'Lucas' }
     let(:email) { 'someemail@example.com' }
     let(:password) { 'validpassword' }
     let(:birth) { Date.new(1990, 11, 0o1) }
     let(:username) { 'lucasuru' }
+    let(:bio) { '' }
+    let(:website) { '' }
```

Comment:
You could use the user factory to generate valid and random attributes for the user. I always try to generate them with the factory and/or use `Faker` so you are testing with different values each time you run the test.